tekton-catalog/pipeline-loops/Makefile

```diff
@@ -0,0 +1,32 @@
+BIN_DIR=_output/bin
+
+# If tag not explicitly set in users default to the git sha.
+TAG ?= v0.0.1
+
+.EXPORT_ALL_VARIABLES:
+
+all: local
+
+init:
+	mkdir -p ${BIN_DIR}
+
+local: init
+	go build -o=${BIN_DIR}/pipelineloop-controller ./cmd/controller
+	go build -o=${BIN_DIR}/pipelineloop-webhook ./cmd/webhook
+
+build-linux: init
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o=${BIN_DIR}/pipelineloop-controller ./cmd/controller
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o=${BIN_DIR}/pipelineloop-webhook ./cmd/webhook
+
+images: build-linux
+	docker build --build-arg bin_name=pipelineloop-controller . -t fenglixa/pipelineloop-controller:$(TAG)
```

Comment:
small nit: change `fenglixa` to a variable (like the tag), you can default it to something generic like `registryname`

Reply:
Thanks @Tomcli , I addressed your comments.